templates/addEvent.phtml

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Add Event</title>
+        <link rel="stylesheet" type="text/css" href="css/genericStyles.css"/>
+        <link rel="stylesheet" href="css/addEventPage.css">
+    </head>
+    <body>
+        <main class="addEventPageWrapper">
+            <h2 id="heading">Add Event</h2>
+            <form action="/addEvent" method="POST" id="addEventForm">
+                <div class="eventDataRow">
+                    <label for="eventName" class="eventLabel">Event Name: </label>
+                    <input type="text" name="eventName" id="eventName" class="eventInput">
+                </div>
+                <div class="eventDataRow">
+                    <label for="location" class="eventLabel">Location: </label>
+                    <input type="text" name="location" id="location" class="eventInput">
+                </div>
+                <div class="eventDataRow">
+                    <label for="eventType" class="eventLabel">Type of Event: </label>
+                    <input type="text" name="eventType" id="eventType" class="eventInput">
```

Comment:
Should this be a select dropdown?

Reply:
It should be, solved in feb0217

Reply:
The options should come from the database.  If you feel this is not part of your task consider creating a new task.